docs/ldml/tr35-keyboards.md

```diff
@@ -252,9 +260,15 @@ If it becomes necessary in the future, the format could extend the ISO layout to
 
 **Virtual keyboard:** see **Touch keyboard**
 
+## Notation
+
+- Ellipsis (`…`) in syntax examples are used to denote substituted parts.
+
+  For example, `id="…keyId"` denotes that `…keyId` (the part between double quotes) is to be replaced with something, in this case a key identifier. As another example, `\u{…}` denotes that the `…` is to be replaced with something, in this case a hex number.
```

Comment:
```suggestion
  For example, `id="…keyId"` denotes that `…keyId` (the part between double quotes) is to be replaced with something, in this case a key identifier. As another example, `\u{…usv}` denotes that the `…usv` is to be replaced with something, in this case a Unicode scalar value in hex.
```

I wonder if we should be explicit here -- it's not UTF-16 or other encoding.

Reply:
good.. should propagate to other sections also since this is just an example